episodes/practical1.Rmd

```diff
@@ -0,0 +1,441 @@
+
+![Figure. ITK-SNAP Reorient image](fig/itk-snap-tools-reorient-image.png)
+
+### 2.4. Navigating DICOM images
+* Navigate around the image and check the intensity value on the cursor inspector.
+* To navigate around the image, you can left-click and drag. To move through the displayed slices, you can use the mouse wheel (or two fingers on the touchpad) or use the slider next to the displayed slice.
+* To zoom, you can use the right click and drag. You can move the entire image by using the mouse centre button or alt + left click and drag.
+
+![Figure. ITK-SNAP cursor inspector](fig/itk-snap-cursor-inspector.png)
+
+### 2.5. Handling multiple DICOM images
+We provide instructions for handling multiple DICOM images, applying overlays, viewing image information, and using colour maps in ITK-SNAP.
+For this example, we use `CT-PET-VI-02/inhale_BH_CT` and `CT-PET-VI-02/exhale_BH_CT` DICOMs.
+
+#### Load additional image
+Go to the `exhale_BH_CT` folder in your File Explorer and drag any DICOM files onto the ITK-SNAP window. 
+A small window will pop up asking what should ITK-SNAP do with this image. 
+If you select `Load as Main Image`, the current image will be replaced. 
+Instead, select `Load as Additional Image` - this will enable easy comparison of the two images.
+
+![Figure. ITK-SNAP additional image](fig/itk-snap-additional-image.png)
+
+* Once the second image is loaded, you will see two thumbnails in the top corner of each slice display
+* Click on these to swap the displayed slices from one image to the other, allowing you to easily see the similarities and differences between the images.
+* You will also see that both images now appear in the Cursor Inspector, where you can see the intensity value at the cursor for both images. 
+* You can also change the displayed image by clicking on it in the Cursor Inspector
+
+![Figure. ITK-SNAP thumbnails, handling two DICOM images (`inhale_BH_CT` and `exhale_BH_CT` for `CT-PET-VI-02`)](fig/itk-snap-thumbnails.png)
+
+#### Colour overlay
+Sometimes, you may want to display an image as a colour overlay on top of another image, e.g., a PET image overlaid on the corresponding CT.
+
+##### Colour overlay as a separate image (shown beside other images)
+* **Load Primary Image**:
+   - Open your primary DICOM series of the `CT-PET-VI-02/CT_for_PET` image either using the drag and drop method or `File -> Open Image` as described above.
+	- Make sure the File Format is DICOM Image Series.
+	- This time, select Load as Main Image. This will load in the new image, replacing the two that were previously loaded.
+* **Load Overlay Image**:
+   - Load the `CT-PET-VI-02/PET` image either using the drag and drop method or `File -> Open Image`
+
+![Figure. ITK-SNAP thumbnails](fig/itk-snap-colour-overlay-as-separate-image.svg)
```

Comment:
kind of - it seems that setting the colourmap here is intended only for use when loading the image as an overlay - if you leave 'as a separate image' selected and change the colourmap it has no effect.
but... if you select 'as an overlay' and then change the colourmap, it will change the colourmap used for the image - then, before pressing next, change the selection back to 'as a separate image' the colourmap will now be used for the separate image as well. but my guess is this is an unintended feature, and the intention is that the colourmap will only be used when loading the image as an overlay, so that's how i will present it in the tutorial.
tbh, the bits on loading multiple images are the bits that i have most heavily modified, as they were not very clear